app/iht/controllers/application/assets/insurancePolicy/InsurancePolicyOverviewController.scala

```diff
@@ -102,22 +102,17 @@ trait InsurancePolicyOverviewController extends EstateController {
   private def createSection1(regDetails: RegistrationDetails, insurancePolicy: InsurancePolicy)(implicit messages:Messages): Section = {
     createSectionFromYesNoValueQuestions(
       id = "deceased",
-      title = Some(messages("iht.estateReport.assets.insurancePolicies.payingOutToDeceased",
-        CommonHelper.getDeceasedNameOrDefaultString(regDetails, true))),
+      title = Some(messages("iht.estateReport.assets.insurancePolicies.payingOutToDeceased", CommonHelper.getDeceasedNameOrDefaultString(regDetails, true))),
```

Comment:
IS the content correct for the insurance policies overview links? Other links have the form: "Give an answer for did the deceased make any changes to a private pension in the 2 years before they died" but here the form is: "Did John Smith(M/CP) have any policies in their own name that were to pay out to them? Give an answer."